src/Easy.zig

```diff
@@ -465,7 +465,7 @@ pub fn setCommonOpts(self: Self) !void {
             .len = bundle.items.len,
             .flags = c.CURL_BLOB_NOCOPY,
         };
-        try checkCode(c.curl_easy_setopt(self.handle, c.CURLOPT_CAINFO_BLOB, blob));
```

Comment:
IMO this is not the right solution.

If we pass a pointer here, after `setCommonOpts` `blob` will be freed, and the pointer will be a dangling pointer?

Reply:
Could you give more context how we can reproduce your issue?

Reply:
Thanks for clarifying, it seems the [examples used in docs](https://curl.se/libcurl/c/CURLOPT_CAINFO_BLOB.html) also pass in pointer.

So it's zig to convert struct to pointer automatically before?

Reply:
Just as an update I haven't managed to figure out why this works on macOS but not on my raspberry pi, and won't look into it any further. The `curl_easy_setopt` is variadic. Somehow on macOS it's happy to take our blob struct and pass the pointer to curl whereas when compiled for linux it's not.

Reply:
Thanks for update.

```zig
pub extern fn curl_easy_setopt(curl: ?*CURL, option: CURLoption, ...) CURLcode;
```

For anyone interested, this is what zig translate-c output.